config/locales/en.option_descriptions.yml

```diff
@@ -484,3 +484,26 @@ en:
       https.  NOTE: Audience1st will also try to serve your
       favicon.ico from the same directory that contains the stylesheet; if
       no favicon.ico is found there, a generic one will be served.
+
+    allow_recurring_donations: >
+
+      Whether to allow customers to initiate recurring monthly donations.
+
+    default_donation_type: >
+
+      Default donation type to be checked when a customer is on the 'make donation' page.
```

Comment:
clarify to "Whether the default option for donations is one-time or recurring when a customer is on the 'make donation' page."